### DISCUSSION TITLE
make *_issue should include only required files part of tar file

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#1307] [General] [answered]
@vvbandeira 

For example if we are submitting issue for `make cts_issue` is that necessary to add all previous stages `odb` file part of the package?

Only `3_*.odb` is sufficient right to run `cts` stage. Because test case size increase more, recently I seen on of the user uploaded `2GB` file size as test case.

@maliberty  Please share your thoughts as well.

**Comment by vvbandeira** [comment]
[maliberty](https://github.com/maliberty) commented [26 minutes ago](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1303#issuecomment-1662281361)
I have mixed feelings about including a single .odb. It would save size with is a plus. However it often happens that someone says they have a problem in step X but really the issue is earlier in the flow (eg macro placement causing drt problems). Having all the prior .odb can be useful. Perhaps we should have an option for large test cases.

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1303#issuecomment-1662281361

**Reply by vvbandeira** [reply, marked as answer]
Reducing size is good indeed. For small testcases, it won't make much difference. On the other hand, large testcases take longer to run, so having all the `.odb` files would prevent the developer from re-running previous stages.

Larger `tar` files reduce developer effort, but makes difficult to share the files.

Smaller `tar` files improve "sharability" but might add time/effort to reproduce issues.

**Reply by vijayank88** [reply]
Considering developer point of view, it make sense.
